kafka-producer/src/test/resources/base.yaml

```diff
@@ -1,29 +1,10 @@
 haystack:
```

Comment:
Is this required after resource/config one?

Reply:
yes , it is required to support backward compatibility.

Reply:
we can discuss this